package.json

```diff
@@ -44,6 +44,7 @@
   },
   "dependencies": {
     "adhan-extended": "^6.1.0",
-    "luxon": "^3.3.0"
+    "luxon": "^3.3.0",
+    "@photostructure/tz-lookup": "^8.0.0"
```

Comment:
how good is this library? in mobile we use another library. please check and compare both and let's use the better one.

Reply:
It is a up to date fork of tz-lookup since it is no longer maintained by darkskyapp
https://github.com/photostructure/tz-lookup